translations/en/course-structure.json

```diff
@@ -0,0 +1,295 @@
+[
```

Comment:
@johnanthos we can probably merge all translations into a single file with simple key/value pairs. We shouldn't have a course structure in each language directory. Rather, we simply have the items that need to be translated in the translation file. E.g.: 

```
{
  "Introduction": "Introducción",
  "Getting started": "Empezando",
  "Explain what web3 is": "Explicar lo que ese Web3",
...
}
```

@mikemaccana any chance you can confirm that this is the right approach for localization?

Reply:
Since not all content is translated, my thought process here was that we could prevent not showing that if we had different structures. Happy to do the reverse and centralize but if we do so it should be by keys / slugs and not full english titles since that may change over time. Hope that makes sense.

Reply:
@jamesrp13 adjusted the PR with the new structure. Let me know what you think.

Reply:
Thanks @johnanthos , looks like you need to pull from `draft` as the course-structure was updated a few weeks ago